src/components/Question.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import D3PieChart from './ReactExample';
+
+const Question = ({ id, question, sideOne, sideTwo, handleSideOneIncrease, handleSideTwoIncrease }) => {
+  return (
+    <section className='current-question'>
+      <div className='piechart-container'>
+        <D3PieChart className='piechart' sideOne={sideOne} sideTwo={sideTwo} />
+        <button onClick={e => handleSideOneIncrease(id)}>I'm Right</button>
```

Comment:
This is highlighted oddly - it might be the ' ?? JSX doesn't like it. Kelly was having issues with it, too. It might be worth just using ```&#39;```